.pre-commit-config.yaml

```diff
@@ -67,6 +67,18 @@ repos:
   hooks:
   - id: black
 
+# - repo: https://github.com/charliermarsh/ruff-pre-commit
```

Comment:
disabled on purpose?

Reply:
Yes, I would like to enable in a following PR